src/apps/content-editor/src/app/views/ItemEdit/ItemEdit.js

```diff
@@ -176,8 +176,21 @@ export default connect((state, props) => {
         .dispatch(fetchItem(modelZUID, itemZUID))
         // select lang based on content lang
         .then(res => {
+          if (res.status === 400) {
+            this.props.dispatch(
+              notify({
+                message: `Failure to edit item: ${res.error}`,
```

Comment:
Lets rephrase this as it wasn't a "failure to edit item" on the users behalf, I could see it read as such. Lets say "Failed saving item: ..."